.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+# trailing whitespace can be significant in markdown
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Throwing in HTML tags for no apparent reason is very ugly. And personally I've never seen anybody using `<br/>` instead of the two tailing whitespaces (which is not GitHub-only but part of the original spec).

I don't want to imply we *need* to use those kind of breaks (there are rather uncommon in most GitHub READMEs), I'm just saying *if* you use them for whatever reason, an `.editorconfig` file with `trim_trailing_whitespace = true` *will* mess it up and you won't find out why unless you're explicitly looking into the config.

At the moment, there are no whitespace induced breaks in any of the Markdown files in this repo. So why not go ahead and use the config as proposed. If we see this really becomes an issue, we can remove the line from `.editorconfig` later.